full_cluster_stop_start_test.py

```diff
@@ -0,0 +1,37 @@
+import time
+
+from sdcm.tester import ClusterTester
+
+
+class FullClusterStopStart(ClusterTester):
+
+    def test_full_cluster_stop_start(self):
+        start_time = time.time()
+        stress_queue = self.run_stress_thread(stress_cmd=self.params.get('stress_cmd'))
+        self.get_stress_results(queue=stress_queue)
+        self.verify_stress_thread(cs_thread_pool=stress_queue)
+        # verify read
+        stress_read = self.run_stress_thread(stress_cmd=self.params.get('stress_read_cmd'))
+        self.verify_stress_thread(cs_thread_pool=stress_read)
+        # stop all nodes
+        nodes = self.db_cluster.nodes
+        for node in nodes:
+            configure_service_command = r"sudo sed -i '/^\[Service\]/a\RestartSec=40sec' /usr/lib/systemd/system/scylla-server.service"
+            demon_reload_command = "sudo systemctl daemon-reload"
```

Comment:
why setting these inside the for_loop every time?
Move it outside the for loop.

Reply:
done